plasma_framework/contracts/src/exits/payment/controllers/PaymentProcessInFlightExit.sol

```diff
@@ -68,7 +73,12 @@ library PaymentProcessInFlightExit {
 
```

Comment:
`PaymentProcessInFlightExit.run()` has several places (basically all `call`s) that would allow for reentrancy. We should refactor the function so it follows Checks-Effects-Interactions pattern -> https://solidity.readthedocs.io/en/develop/security-considerations.html?highlight=check%20effects#use-the-checks-effects-interactions-pattern